modules/jooby-apt/src/main/java/io/jooby/apt/JoobyProcessor.java

```diff
@@ -51,26 +36,28 @@
     Opts.OPT_DEBUG,
     Opts.OPT_INCREMENTAL,
     Opts.OPT_SERVICES,
-    Opts.OPT_SKIP_ATTRIBUTE_ANNOTATIONS })
+    Opts.OPT_SKIP_ATTRIBUTE_ANNOTATIONS})
 public class JoobyProcessor extends AbstractProcessor {
 
   private ProcessingEnvironment processingEnv;
 
   /**
    * Route Data.
    * {
-   *   HTTP_METHOD: [method1, ..., methodN]
+   * HTTP_METHOD: [method1, ..., methodN]
    * }
    */
   private Map<TypeElement, Map<TypeElement, List<ExecutableElement>>> routeMap = new LinkedHashMap<>();
 
   private boolean debug;
   private boolean incremental;
   private boolean services;
+  private boolean inspectSuperTypes;
```

Comment:
we need to find another name to better the describe it. Inherited does work as long as classes are on same project (as you already figured it out).

Reply:
HI @jknack I have renamed it to inspectSubClasses as its really what it does

Reply:
Still found a bit confusing, bc inherited works without this option enabled. The issue raises when class isn't part of the project (source code)